clients/admin-ui/src/features/privacy-notices/table/usePrivacyNoticesTable.tsx

```diff
@@ -0,0 +1,226 @@
+            >
+              {record.name}
+            </LinkCell>
+          ),
+        },
+        {
+          title: "Mechanism",
+          dataIndex: "consent_mechanism",
+          key: "consent_mechanism",
+          render: (value: ConsentMechanism) => <MechanismCell value={value} />,
+        },
+        {
+          title: "Locations",
+          dataIndex: "configured_regions",
+          key: "regions",
+          render: (regions: PrivacyNoticeRegion[] | undefined) => {
+            const values =
+              regions?.map((location: PrivacyNoticeRegion) => {
+                const isoEntry = isoStringToEntry(location);
+                return {
+                  label: isoEntry
+                    ? formatIsoLocation({ isoEntry, showFlag: true })
+                    : (PRIVACY_NOTICE_REGION_RECORD[location] ?? location),
+                  key: location,
+                };
+              }) ?? [];
+            return (
+              <TagExpandableCell
+                values={values}
+                columnState={{
+                  isExpanded: isLocationsExpanded,
+                }}
+              />
+            );
+          },
+          menu: {
+            items: expandCollapseAllMenuItems,
+            onClick: (e) => {
+              e.domEvent.stopPropagation();
+              if (e.key === "expand-all") {
+                setIsLocationsExpanded(true);
+              } else if (e.key === "collapse-all") {
+                setIsLocationsExpanded(false);
+              }
+            },
+          },
+        },
+        {
+          title: "Status",
+          dataIndex: "disabled",
+          key: "status",
+          render: (_: boolean, record: PrivacyNoticeRowType) => (
+            <StatusCell record={record} />
+          ),
+        },
+        {
+          title: "Framework",
+          dataIndex: "framework",
+          key: "framework",
+          render: (framework: string | null) =>
+            framework ? (
+              <Tag data-testid="framework-badge">
+                {FRAMEWORK_MAP.get(framework) ?? framework}
+              </Tag>
+            ) : null,
+        },
+        {
+          title: "Children",
+          dataIndex: "noticeChildren",
+          key: "noticeChildren",
+          render: (noticeChildren?: LimitedPrivacyNoticeResponseSchema[]) => (
+            <NoticeChildrenCell value={noticeChildren} />
+          ),
+        },
+        ...(userCanUpdate
+          ? [
+              {
+                title: "Enable",
+                dataIndex: "disabled",
+                key: "enable",
+                render: (_: boolean, record: PrivacyNoticeRowType) => (
+                  <NoticeEnableCell record={record} />
+                ),
+                onCell: () => ({
+                  onClick: (e: React.MouseEvent) => e.stopPropagation(),
+                }),
+              },
+            ]
+          : []),
+      ] as ColumnsType<PrivacyNoticeRowType>,
```

Comment:
Shouldn't cast here.
```suggestion
      ],
```